lib/pageflow/configuration.rb

```diff
@@ -476,5 +488,27 @@ class FeatureLevelConfiguration < Struct.new(:config)
       delegate :themes, to: :config
       delegate :file_importers, to: :config
     end
+
+    module EntryTypeConfiguration
+      def assign_config(value)
+        @config = value
+      end
+
+      def config
+        @config ||= Configuration.new
+      end
+
+      def method_missing(method, *args)
+        if @config.respond_to?(method)
+          config.send(method, *args)
+        else
+          super
+        end
+      end
+
+      def respond_to_missing?(method_name, include_private = false)
+        @config.respond_to?(method_name) || super
+      end
```

Comment:
I'd suggest to directly start with a restricted interface here, which only delegates selected methods to the main configuration. `config_for` then needs to return the delegator we talked about, which delegates missing methods to the entry type specific config.